faqs/what-is-the-merge.md

```diff
@@ -3,6 +3,7 @@ title: What is "The Merge"?
 weight: 1.0
 attribution:
   -
-    name:
-    link:
----
\ No newline at end of file
+    name: Lamboshi
+    link: https://twitter.com/L_Nakaghini
+---
+The Merge is an upgrade to Ethereum that swaps out the current proof of work consensus mechanism with a more effecient and secure proof of stake consensus mechanism. This means that the current proof of work consensus mechanism is deprecated, and all blocks on eteherum will be produced by stakers, not miners. 
```

Comment:
Great to see you! My only feedback for this one would be to add the PoS and PoW abbreviations in parenthesis next to their full written names so that news users start to absorb the lingo. 👍 

Might also be useful to add a sentence briefly about the benefits of the merge - namely energy efficiency - which is going to be a more and more important criticism to address as time marches on.